test/unit/specs/components/governance/LiProposal.spec.js

```diff
@@ -86,29 +140,16 @@ describe(`LiProposal`, () => {
   })
 
   it(`should truncate the description and add an ellipsis`, () => {
-    proposal.description = `this is some kind of long description. longer than 100 characters for optimum-maximum-ideal truncation.`
-    wrapper.setProps({ proposal: JSON.parse(JSON.stringify(proposal)) })
-    expect(wrapper.vm.description).toEqual(
-      `this is some kind of long description. longer than 100 characters for optimum-maximum-ideal truncati…`
-    )
-  })
-
-  it(`survives the tally result not being present yet`, () => {
```

Comment:
Why did you kill this test?

Reply:
it wasn't testing anything

Reply:
it probably tested, that it wouldn't throw an error.

Reply:
ok i added a test back.